sdk/eventgrid/eventgrid/README.md

```diff
@@ -174,6 +174,10 @@ When creating an instance of `EventGridDeserializer` you may supply custom deser
 
 This library supports distributed tracing using [`@azure/core-tracing`][azure-core-tracing-github]. When using distributed tracing, this library will create a span during a `send` operation. In addition, when sending events using the Cloud Events 1.0 schema, the SDK will add distributed tracing metadata to the events using the [Distributed Tracing extension][cloud-events-distributed-tracing-spec]. The values for the `traceparent` and `tracestate` extension properties correspond to the `traceparent` and `tracestate` headers from the HTTP request which sends the events. If an event already has a `traceparent` extension property it is not updated.
 
+### Event Grid on Kubernetes
+
+This library has been tested and validated on Kubernetes using Azure Arc.
```

Comment:
I know nothing about kubernetes, but is there any version # applicable here?

Reply:
Sadly no, but I think I can make "Kubernetes using Azure Arc" a link to https://docs.microsoft.com/en-us/azure/event-grid/kubernetes/ which is the service documentation on this new feature (it's something in preview that they launched at build, the data-plane API is the same as the version hosted in Azure so our SDK just works with it and we went thought the validation exercise on our end to confirm that).